app/src/main/AndroidManifest.xml

```diff
@@ -37,7 +37,7 @@
         android:label="@string/app_name"
         android:launchMode="singleInstance"
         android:theme="@style/Theme.MyTheme"
-        tools:replace="android:appComponentFactory"
+        tools:replace="android:appComponentFactory,android:theme"
```

Comment:
why you add `android:theme `here ?

Reply:
ping @dhouha123

Reply:
it was requested in the first version but we no longer need for the last implementation